Tests/test_specific_mechanisms.py

```diff
@@ -0,0 +1,119 @@
+import pytest
+"""
+A catch all of tests to ensure different mechanisms behave as desired
+"""
+
+from biocrnpyler import Species, Complex
+
+#Test Metabolite Mechanisms
+from biocrnpyler import OneStepPathway
+
+def test_OneStepPathway():
+    osp = OneStepPathway()
```

Comment:
why the naming convention here is not with underscores like One_Step_Pathway like for other mechanisms? or the other way round for other mechanisms?

Reply:
I think we have not been very consistent with our naming convention, unfortunately.